### ISSUE TITLE
Task :react-native-cardview:verifyReleaseResources FAILED

### ISSUE THREAD

**Issue by kamlesh1000** [Kishanjvaghela/react-native-cardview#52]
FAILURE: Build failed with an exception.

* What went wrong:
```
Execution failed for task ':react-native-cardview:verifyReleaseResources'.
> 1 exception was raised by workers:
  com.android.builder.internal.aapt.v2.Aapt2Exception: Android resource linking failed
  C:\Users\Kamlesh\Desktop\ControlPlus\node_modules\react-native-cardview\android\build\intermediates\res\merged\release\values-v28\values-v28.xml:7: error: resource android:attr/dialogCornerRadius not found.
  C:\Users\Kamlesh\Desktop\ControlPlus\node_modules\react-native-cardview\android\build\intermediates\res\merged\release\values-v28\values-v28.xml:11: error: resource android:attr/dialogCornerRadius not found.
  C:\Users\Kamlesh\Desktop\ControlPlus\node_modules\react-native-cardview\android\build\intermediates\res\merged\release\values\values.xml:2788: error: resource android:attr/fontVariationSettings not found.
  C:\Users\Kamlesh\Desktop\ControlPlus\node_modules\react-native-cardview\android\build\intermediates\res\merged\release\values\values.xml:2789: error: resource android:attr/ttcIndex not found.
  error: failed linking references.
```
* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

Deprecated Gradle features were used in this build, making it incompatible with Gradle 6.0.
Use '--warning-mode all' to show the individual deprecation warnings.
See https://docs.gradle.org/5.4.1/userguide/command_line_interface.html#sec:command_line_warnings

IN DEBUG MODE:
```
WARNING: Configuration 'compile' is obsolete and has been replaced with 'implementation' and 'api'.
It will be removed at the end of 2018. For more information see: http://d.android.com/r/tools/update-dependency-configurations.html
WARNING: The specified Android SDK Build Tools version (27.0.3) is ignored, as it is below the minimum supported version (28.0.3) for Android Gradle Plugin 3.4.2.
Android SDK Build Tools 28.0.3 will be used.
To suppress this warning, remove "buildToolsVersion '27.0.3'" from your build.gradle file, as each version of the Android Gradle Plugin now has a default version of the build tools.
```

React-native Environment:
```
System:
    OS: Windows 10
    CPU: (4) x64 Intel(R) Core(TM) i5-5200U CPU @ 2.20GHz
    Memory: 2.41 GB / 7.92 GB
  Binaries:
    Node: 12.3.1 - C:\Program Files\nodejs\node.EXE
    Yarn: 1.16.0 - C:\Program Files (x86)\Yarn\bin\yarn.CMD
    npm: 6.9.0 - C:\Program Files\nodejs\npm.CMD
  IDEs:
    Android Studio: Version  3.4.0.0 AI-183.6156.11.34.5522156
  "react": "16.8.6",
  "react-native": "0.60.4",
    "react-native-cardview": "^2.0.2",
```

**Comment by Kishanjvaghela** [comment]
Published to v2.0.3
https://www.npmjs.com/package/react-native-cardview
Feel free to reopen the issue if not working.

**Comment by club9822** [comment]
if you don't migrated to `androidx`  do following:
npm i ... v2.0.3 with `npx jetify -r` to convert to support lib then change  
`compile 'androidx.cardview:cardview:1.0.0'  to  `implementation"com.android.support:cardview-v7:${rootProject.ext.supportLibVersion}"` in `build.gradle` of this module 

"react": "^16.8.3",
 "react-native": "^0.59.10",
 "react-native-cardview": "^2.0.3"
